terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ManageReservationHelper.java

```diff
@@ -135,7 +140,106 @@ public ReservationDetailOutput findDetail(ManageReservationForm form) {
     }
 
     public DownloadPDFOutput createPDF(String reserveNo) {
+        ReservationDetailOutput reserveDetailOutput = findDetail(reserveNo);
+
+        // assigning display string to conductor on the basis of DB value
+        String conductor = null;
+        if ("1".equals(reserveDetailOutput.getReserve().getTourInfo()
+                .getConductor())) {
+            conductor = getMessage(MessageId.LABEL_TR_COMMON_YESMESSAGE);
+        } else {
+            conductor = getMessage(MessageId.LABEL_TR_COMMON_NOMESSAGE);
+        }
+
+        String paymentTimeLimit = null;
+        if ("1".equals(reserveDetailOutput.getReserve().getTransfer())) {
+            paymentTimeLimit = getMessage(MessageId.LABEL_TR_MANAGERESERVATION_DONE);
+        } else {
+            // TODO use propertyUtil to fetch the format

```

Comment:
* please remove TODO comment if unnecessary.
* please use API of Joda-Time instead of `SimpleDateFormat`